libs/s25main/network/GameClient.h

```diff
@@ -300,6 +310,10 @@ class GameClient final :
 
     std::unique_ptr<ReplayInfo> replayinfo;
     bool replayMode;
+
+    // Configured players for an AI battle.
```

Comment:
Docstrings need to start with 3 slashes

```suggestion
    /// Configured players for an AI battle.
```

Reply:
agreed